wgpu/src/backend/web.rs

```diff
@@ -1123,6 +1124,8 @@ impl crate::context::Context for Context {
             // Doesn't really have meaning on the web.
             present_modes: vec![wgt::PresentMode::Fifo],
             alpha_modes: vec![wgt::CompositeAlphaMode::Opaque],
+            
+            // Something has to go here
```

Comment:
surface handling on web is a bit shacky - the spec doesn't have this api as-is as it works with the canvas directly, so it's more of a pure wgpu construct.
Only the RENDER_ATTACHMENT usage should go here for now - see also https://www.w3.org/TR/webgpu/#canvas-configuration

Reply:
Well that makes my job easy atleast ^^
Perhaps in the future this can provide the usages https://www.w3.org/TR/webgpu/#dom-gpucanvascontext-getcurrenttexture. I saw it was unstable for now though